apps/aws-notebook/README.md

```diff
@@ -0,0 +1,10 @@
+# Application For Jupter Notebook
+
+## Parameter In Environment Variables
+```
+AWS_CONFIG_FILE: the path of config file for aws cli
+OPENVPN_CONFIG_PATH: the path of config file for openvpn 
+ECS_CLUSTER: the name of the ecs cluster
+SUBNET_ID: which subnet the service task will use
+SERVICE_LAUNCH_TYPE: FARGATE or EC2
+```
```

Comment:
A better way to write markdown:

```markdown
## Parameters In Environment Variables

- `AWS_CONFIG_FILE`: the path to config file for AWS CLI stored as a secret;
- `OPENVPN_CONFIG_FILE`: the path to the OpenVPN config file stored as a secret to connect to the private VPC;
- `ECS_CLUSTER`: the name of the ECS cluster;
- `SUBNET_ID`: the subnet to create ECS tasks;
- `SERVICE_LAUNCH_TYPE`: `FARGATE` or `EC2`
```

Reply:
Please also sort the environment variables: the mandatory ones come first, optional ones come later. And if it's optional, please specify explicitly, what's the default value.

I also changed `OPENVPN_CONFIG_PATH` to `OPENVPN_CONFIG_FILE` just to be consistent.